python/src/lib.rs

```diff
@@ -217,6 +271,7 @@ impl<T: Ipc> CongAlg<T> for PyAlg {
             }
         };
     }
+    // TODO implement close, deallocate memory from class
```

Comment:
Did this need to be done?

Reply:
doesn't need to be done, everything works without it, but I want to look into it more later in case it should be done. this is included in #51

Reply:
in that case what are these TODO methods for?